Fix overflow before widen

expliyh opened the pull request:
bug: v/76743

Rootcause: 2 * stream->interval_ms * 1000 will use 32-bit integer when calculate on a 32-bit platform.

gzh-terry commented on service/profiles/a2dp/source/a2dp_source_audio.c:
```diff
@@ -252,7 +252,7 @@ static void a2dp_source_audio_handle_timer(service_timer_t* timer, void* arg)
     uint64_t now_us = bt_get_os_timestamp_us();
-    if (stream->last_ts && ((now_us - stream->last_ts) > (2 * stream->interval_ms * 1000))) {
+    if (stream->last_ts && ((now_us - stream->last_ts) > (2ULL * (uint64_t)stream->interval_ms * 1000ULL))) {
         BT_LOGD("===a2dp cpu busy time:%lld===", now_us - stream->last_ts);
```
Can you help change this to
```
BT_LOGD("===a2dp cpu busy time:%" PRIu64 "===", now_us - stream->last_ts);
```

expliyh replied:
This is a Cherry-Pick. It is recommended to use a different patch instead.

expliyh committed:
Fix overflow before widen

bug: v/76743

Rootcause: 2 * stream->interval_ms * 1000 will use 32-bit integer when calculate on a 32-bit platform.

Signed-off-by: liyuheng <liyuheng@xiaomi.com>
